code/headers/mlx_parameters.hpp

```diff
@@ -0,0 +1,57 @@
+#pragma once
+#include <cstdint>
+
+namespace r2d2::thermal_camera {
+    struct mlx_parameters_s {
+        /**
+         * Supply voltage in Volts
+         */
+        static constexpr float VDD0 = 3.3;
+        /**
+         * TA0, datasheets refers to 25.
+         */
+        static constexpr uint8_t TA0 = 25;
+        /**
+         * Emissivity compensation: We assume Emissivity = 1.
+         * Emissivity coefficient is user defined and it is not stored in the
+         * device EEPROM)
+         */
+        float emissivity;
+        // Kvdd, required for calculations
+        int Kvdd;
```

Comment:
Please either explain what the value is, or reference the datasheet with a page number

Reply:
Same goes for the other variables.

Reply:
Please see the pipeline `mlx90640_processor_c::get_temperature_pixel`. Each object is (approximately) responsible for its own calculation. Above the creation of each object in `mlx90640_processor_c::get_temperature_pixel`, there is a reference to a section in the datasheet, in which these numbers come back.